### PR TITLE
Removes occurrences of $.browser

### LINKED ISSUE #510: Stop using $.browser
It was removed in jQuery 1.9, so this makes using a custom jQuery impossible..

http://api.jquery.com/jquery.browser/

### PR DESCRIPTION
**Pull request by fdansv**
Closes #510

### CODE REVIEW

**Comment by javisantana** [comment]
I don't like the approach, I rather use:

- an explicit browser detection. For example, in the infowindow animation we disable IE because of performance. If we are going with those tricks, better to hide it behind a helper

- explicit feature detection for fullscreen for example

**Comment by fdansv** [comment]
@javisantana added `cdb.core.util.browser`. Tested with all desktop browsers and all IE js engines from IE6 upwards.

File: src/core/util.js [hunk 1]
```diff
@@ -87,4 +87,37 @@ cdb.core.util._makeCRCTable = function() {
...
+  };
+
+  if(detectIE()){
+    browser.ie = {version: getIEVersion()}
+  }
+
+  else if(ua.indexOf('Chrome') > -1) browser.chrome = ua;
+  else if(ua.indexOf('Firefox') > -1) browser.firefox = ua;
+  else if(ua.indexOf("Safari") > -1) browser.safari = ua;
+  else if(ua.toLowerCase().indexOf("op") > -1) browser.opera = ua;
```
Comment (javisantana) [inline src/core/util.js]: why in this case is different?

Comment (fdansv) [inline src/core/util.js]: Whoops, that's the first one I added, forgot to set it like the rest. Correcting it now.

File: test/spec/core/util.spec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+fdescribe("util", function() {
```
Comment (javisantana) [inline test/spec/core/util.spec.js]: is this fdescribe ok?

Comment (fdansv) [inline test/spec/core/util.spec.js]: no, but it's changed now

**Pull request merged**